config.js

```diff
@@ -24,5 +26,20 @@ module.exports = {
 		templates: "client/hbs",
 		upload: "upload"
 	},
+	categories:[
```

Comment:
The way the frontend is written, we don't need to explicitly have a list of categories.  They're pulled from the list across all posts.